packages/terra-heading/docs/heading-colors.md

```diff
@@ -1,3 +1,28 @@
 ## Heading Colors
 
-Using the `color` prop, authors can set the text color of any `<Heading />` component using a valid CSS color value.
+By passing in a className, authors can manually set the text color of any `<Heading />` component.
+
+```css
+:root {
+  --terra-heading-attention-color: rgb(220, 20, 60);
```

Comment:
The `--terra` prefix is intended to be used only by components we write in terra. Since this example is intended for people to write in their own app, we should avoid using the `--terra` prefix. I'd recommend dropping the prefix or use a prefix like `--my-app`.

Reply:
Okay mostly these were there, because in the actual css files used for the examples, if I used custom variables it linter complained about not starting with the appropriate prefix and suffix. Though we can easily make it so that the docs use `--my-app` and the actual css file uses `--terra`

Reply:
Yeah let's do that.👍

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/0cb2f32c1a47112a45feb6c38ab703cbc70c40ca